src/Analyzers/CSharp/Tests/UseCollectionExpression/UseCollectionExpressionForStackAllocTests.cs

```diff
@@ -58,6 +56,48 @@ void M()
                 }
                 """,
             LanguageVersion = LanguageVersion.CSharp11,
+            ReferenceAssemblies = ReferenceAssemblies.Net.Net80,
+        }.RunAsync();
+    }
+
+    [Fact]
+    public async Task TestNotInCSharp12_Net70()
+    {
+        await new VerifyCS.Test
+        {
+            TestCode = """
+                using System;
+
+                class C
+                {
+                    void M()
+                    {
+                        ReadOnlySpan<int> x = stackalloc int[] { 1, 2, 3 };
```

Comment:
❓ Can you provide more information about this one? I would expect it to still work for blittable types.

Reply:
that's coming in a followup PR *once* compiler implements that optimization.  We don't have the optimization *yet*, so i don't want the features doing something suboptimal in the meantime :)